routes/api.php

```diff
@@ -16,4 +17,13 @@
 
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
-});
\ No newline at end of file
+});
+
+Route::post('register',[AuthController::class, 'register']);
+Route::post('login',[AuthController::class, 'login']);
+
+Route::group(['middleware' => ['auth:sanctum']], function(){
```

Comment:
Create a route group prefixed with "ext" that uses auth:sanctum middleware and has all the stack exchange routes.